<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>司法大数据分析</string>
  </property>
  <widget class="QPushButton" name="loadcase_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>上传案例文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>清空案例文字</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>780</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>保存案件与标注</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>681</width>
     <height>681</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="casetext_edit">
      <property name="plainText">
       <string/>
      </property>
      <property name="placeholderText">
       <string>请输入案件信息</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="party_tab">
       <attribute name="title">
        <string>当事人</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>48</y>
          <width>591</width>
          <height>241</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCheckBox" name="party_choice1">
           <property name="text">
            <string>待选选项</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">party_group</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="party_choice2">
           <property name="text">
            <string>待选选项</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">party_group</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="party_choice3">
           <property name="text">
            <string>待选选项</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">party_group</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="party_choice4">
           <property name="text">
            <string>待选选项</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">party_group</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="gender_tab">
       <attribute name="title">
        <string>性别</string>
       </attribute>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>当事人1</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>当事人2</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>21</x>
          <y>80</y>
          <width>281</width>
          <height>21</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="boy_choice1">
           <property name="text">
            <string>男</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">gender_group1</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="girl_choice1">
           <property name="text">
            <string>女</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">gender_group1</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>200</y>
          <width>271</width>
          <height>21</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QCheckBox" name="boy_choice2">
           <property name="text">
            <string>男</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">gender_group2</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="girl_choice2">
           <property name="text">
            <string>女</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">gender_group2</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="nation_tab">
       <attribute name="title">
        <string>民族</string>
       </attribute>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>当事人1</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>当事人2</string>
        </property>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>21</x>
          <y>90</y>
          <width>641</width>
          <height>21</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="nation_choice1">
           <property name="text">
            <string>待选选项</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">nation_group1</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="nation_choice2">
           <property name="text">
            <string>待选选项</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">nation_group1</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>198</y>
          <width>641</width>
          <height>21</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QCheckBox" name="nation_choice12">
           <property name="text">
            <string>待选选项</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">nation_group2</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="nation_choice22">
           <property name="text">
            <string>待选选项</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">nation_group2</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="birthplace_tab">
       <attribute name="title">
        <string>出生地</string>
       </attribute>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>当事人1</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>当事人2</string>
        </property>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>60</y>
          <width>601</width>
          <height>101</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QCheckBox" name="birthplace_choice1">
           <property name="text">
            <string>待选选项</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">birthplace_group1</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="birthplace_choice2">
           <property name="text">
            <string>待选选项</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">birthplace_group1</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>190</y>
          <width>601</width>
          <height>101</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QCheckBox" name="birthplace_choice12">
           <property name="text">
            <string>待选选项</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">birthplace_group2</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="birthplace_choice22">
           <property name="text">
            <string>待选选项</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">birthplace_group2</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="cause_tab">
       <attribute name="title">
        <string>案由</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>481</width>
          <height>231</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_17">
         <item>
          <widget class="QCheckBox" name="cause_choice1">
           <property name="text">
            <string>待选选项</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">cause_group</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cause_choice2">
           <property name="text">
            <string>待选选项</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">cause_group</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cause_choice3">
           <property name="text">
            <string>待选选项</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">cause_group</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cause_choice4">
           <property name="text">
            <string>待选选项</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">cause_group</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="court_tab">
       <attribute name="title">
        <string>相关法院</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>611</width>
          <height>251</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="court_choice1">
           <property name="text">
            <string>待选选项</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">court_group</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="court_choice3">
           <property name="text">
            <string>待选选项</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">court_group</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="court_choice2">
           <property name="text">
            <string>待选选项</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">court_group</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="court_choice4">
           <property name="text">
            <string>待选选项</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">court_group</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>463</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="findcase_button">
      <property name="text">
       <string>爬取案件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>开始日期</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="startdate"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>结束日期</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="enddate"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="analyse_button">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>60</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>分析案例</string>
   </property>
  </widget>
  <widget class="QPushButton" name="batchprocess_button">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>60</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>批量处理</string>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>loadcase_button</zorder>
  <zorder>clear_button</zorder>
  <zorder>save_button</zorder>
  <zorder>analyse_button</zorder>
  <zorder>batchprocess_button</zorder>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="gender_group2"/>
  <buttongroup name="nation_group2"/>
  <buttongroup name="birthplace_group2"/>
  <buttongroup name="party_group">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
  <buttongroup name="nation_group1"/>
  <buttongroup name="gender_group1"/>
  <buttongroup name="birthplace_group1">
   <property name="exclusive">
    <bool>true</bool>
   </property>
  </buttongroup>
  <buttongroup name="court_group">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
  <buttongroup name="cause_group">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
